reporter-influxdb/src/test/java/io/ultrabrew/metrics/reporters/influxdb/InfluxDBClientTest.java

```diff
@@ -147,6 +147,22 @@ public void testWriteSplitting() throws IOException {
     }};
   }
 
+  @Test
+  public void testNullStrings() throws Exception {
+    new Expectations() {{
+      
```

Comment:
If there are no expectations, just remove the entire block.

Reply:
done